File: app/src/main/java/ru/aradxxx/ciceronetabssample/tabfragment/TabContainerScreen.java
package ru.aradxxx.ciceronetabssample.tabfragment;

import androidx.fragment.app.Fragment;
import ru.terrakok.cicerone.android.support.SupportAppScreen;

/**
 * Пример реализации экрана контейнера табов.
 */
public class TabContainerScreen extends SupportAppScreen {
    @Override
    public Fragment getFragment() {
        return new TabContainerFragment();
    }
}

File: app/src/main/java/com/github/aradxxx/ciceroneflowssample/tabfragment/TabContainerScreen.kt
package com.github.aradxxx.ciceroneflowssample.tabfragment

import com.github.terrakok.cicerone.androidx.FragmentScreen

/**
 * Пример реализации экрана контейнера табов.
 */
class TabContainerScreen : FragmentScreen(
    createFragment = {
        TabContainerFragment()
    }
)
